components/teams-service/storage/postgres/postgres.go

```diff
@@ -258,42 +258,56 @@ func (p *postgres) GetTeams(ctx context.Context) ([]storage.Team, error) {
 // server code before we get here if the team in question was filtered, but if we call this
```

Comment:
Is this comment still accurate? (Two lines up)
> NOTE: We aren't filtering on projects here [...]

Reply:
woops missed that